admin_manual/office/troubleshooting.rst

```diff
@@ -0,0 +1,41 @@
+===============
+Troubleshooting
+===============
+
+In case of connectivity issues, ensure that the following required connections are possible and not blocked by any firewall:
+
+- The users browser can reach both the Nextcloud Server as well as the Nextcloud Office server through HTTP(S)
+- The Nextcloud and the Nextcloud Office Server are using the same protocol
+- The Nextcloud server can reach the Nextcloud Office Server through HTTP(S)
+- The Nextcloud Office Server can reach the Nextcloud server through HTTP(S)
+
+Both the Nextcloud log as well as the Nextcloud Office Server log may reveal more detailed error messages in case of connection issues.
+
+- Verify connectivity from the browser:
+    - https://office.example.com/hosting/capabilities
+    - https://office.example.com/hosting/discovery
+- Verify connectivity from Nextcloud
+    - ``curl https://office.example.com/hosting/capabilities``
+    - ``curl https://office.example.com/hosting/discovery``
+- Verify connection from the Collabora server
+    - ``curl https://nextcloud.example.com/status.php``
+
+Frequently asked questions
+==========================
+
+Issue: I get connection errors when trying to open documents
+    Be sure to check the error log from docker through ``docker logs container-id``. If the logs note something like:
+    ``No acceptable WOPI hosts found matching the target host [YOUR NEXTCLOUD DOMAIN] in config.``
+    Unauthorized WOPI host. Please try again later and report to your administrator if the issue persists. You might have started the docker container with the wrong URL. Be sure to triplecheck that you start it with the URL of your Nextcloud server, not the server where Collabora Online runs on.
```

Comment:
Seems like the quoting is wrong here. Is "Unauthorized WOPI host. Please try again later and report to your administrator if the issue persists." still part of the error log?

Also, here you use "Collabora Online" instead of "Nextcloud Office Server" again ;)